### ISSUE TITLE
Missing `variant` in UniversalFilter component 

### ISSUE THREAD

**Issue by dragon-slayer875** [layer5io/sistent#548]
#### Current Behavior

The Select component used in `UniversalFilter` requires a variant prop which is currently missing and thus gives a typeError.

#### Expected Behavior

A variant prop should be provided in the the props of UniversalFilter that is passed down to the Select.

#### Screenshots/Logs

![image](https://github.com/layer5io/sistent/assets/59254790/b349e378-4421-4925-8b03-35ae42558d21)

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
  - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Sistent site](https://www.figma.com/file/ekm0Hh6M25bSKpP5BcIJom/Sistent-Design-System?type=design&node-id=596%3A5227&mode=design&t=8MExPykHN1vQFCTZ-1) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by dragon-slayer875** [comment]
I am ready with a PR that solves this issue and would like to request an assignment.

**Comment by sudhanshutech** [comment]
i am not getting any such error:
![image](https://github.com/layer5io/sistent/assets/90546692/7bb27b61-1846-4684-b507-fc5b3b4c7c5e)

**Comment by dragon-slayer875** [comment]
Yes. As @nebula-aac stated, the PR fails some build checks because of the missing variant prop @sudhanshutech.

**Comment by dragon-slayer875** [comment]
@sudhanshutech Were you able to confirm the issue?
